toolboxes/core/cpu/hoNDObjectArray.h

```diff
@@ -10,5 +10,57 @@ if delete_data_on_destruct == true, the object will be released; otherwise, only
 
 namespace Gadgetron
 {
-    template<class T> using hoNDObjectArray = hoNDArray<T>;
-}
+    template <typename TObjectType> class hoNDObjectArray : public hoNDArray<TObjectType>
```

Comment:
I don't understand why we need `hoNDObjectArray<T>`, when `hoNDArray<T>` will do. 

What motivates this necromancy?

Reply:
fixed. All are using hoNDArray .